packages/react-composites/review/react-composites.api.md

```diff
@@ -226,8 +232,8 @@ export interface CallCompositeProps extends BaseCompositeProps<CallCompositeIcon
     adapter: CallAdapter;
     // (undocumented)
     callInvitationURL?: string;
+    hiddenElements?: CallCompositeHiddenElements;
```

Comment:
With it being hiddenElements (so everything is shown by default) and not visualElements (where everything has to be set to true explicitly) -what happens if we want to add an optional visual element ?
Would showing it by default be a breaking change? (advantage of visualElements over hiddenElements was that we could add it to this and it would be hidden by default)

Reply:
You have a valid point. Had a discussion around this with @alkwa-msft and decided to make every UI feature visible by default going forward.

Couple of ways to solve that would be to add that new feature to hidden elements with a default `true`

@alkwa-msft any suggestions?

Reply:
The defaulting to true takes away from @prprabhu-ms's main point that `undefined === false` + @prprabhu-ms for ideas here